obal/data/roles/setup-sources/tasks/main.yml

```diff
@@ -0,0 +1,59 @@
+---
+- set_fact:
+    package_dir: "{{ inventory_dir }}/packages/{{ inventory_hostname }}"
+    setup_sources_annex_options: setup_sources_annex_options
```

Comment:
Why is this needed?

Reply:
it's not, esp it sets the `setup_sources_annex_options` fact to the literal `setup_sources_annex_options` string

Reply:
Yeah, I think this snuck in there b/c I had the defaults in the wrong place and I didn't remove it